watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -330,14 +362,14 @@ def build(self):
             units=pyunits.m,
             doc="Stokes radius of solute",
         )
-        self.diffus_phase_comp = Param(
-            self.phase_list,
-            self.ion_set | self.solute_set,
+        self.molar_volume_comp = Param(
+            ["Liq"],
```

Comment:
```suggestion
            self.phase_list,
```
Current use of molar volume in H-L is liquid only but I checked and other phases are valid so I agree.